README.md

```diff
@@ -28,6 +28,47 @@ through Kafka as more ephemeral and want the flexibility to change how we host K
 In the future we may also apply per-subject permissions to the Avro schemas that
 are stored by the registry.
 
+### Extensions
+
+In addition to the Confluent Schema Registry API, this application provides an
+endpoint that can be used to determine by fingerprint if a schema is already
+registered for a subject.
+
+This endpoint provides a success response that can be cached indefinitely since
+the id for a schema will not change once it is registered for a subject.
+
+`GET /subjects/(string: subject)/fingerprints/(:fingerprint)`
```

Comment:
@jturkel I was attempting to be somewhat consistent with the existing schema registry API. None of the standard endpoints use query parameters. Here are the existing endpoints that use GET:

- GET `/schemas/ids/:id` - returns a schema
- GET `/subjects` - returns an array of subject names
- GET `/subjects/:subject/versions` - returns an array of version numbers
- GET `/subjects/:subject/versions/:version` - returns name, version, and schema
- GET `/config` - returns global config
- GET `/config/:subject` - returns subject config

I wanted an endpoint that would return the same result as POST `/subjects/:subject/versions` for a previously registered schema, just the id of the schema. 

Originally I had implemented `/schemas/fingerprints/:fingerprint` but then I realized that it needed to be scoped by subject to be sure that the schema was registered for the subject.